backend/Resources/SaveMetadataCategoryResource.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System;
+
+namespace OpenData.Resources
+{
+	///<summary>
+	///Represents the required fields for the creation of a metadata object
+	///</summary>
+	public class SaveMetadataCategoryResource
+	{
+		[MaxLength(128)]
```

Comment:
MaxLength is used for number of characters allowed. Should be smaller for names.

Reply:
I dont think it reffers to bit-length